tests/create_set1_fixtures.py

```
@@ -0,0 +1,178 @@
+
+    output_path = fixtures_dir / "hyperlinks.docx"
+    doc.save(output_path)
+    print(f"Created: {output_path}")
+
+def create_comments_docx():
+    """Create DOCX with comments."""
+    doc = Document()
+    doc.add_heading("Comments Fixture", 0)
+
+    p = doc.add_paragraph("This is a paragraph with a comment.")
+    run = p.runs[0]
```

Ruff (0.13.1)

Local variable `run` is assigned to but never used

Remove assignment to unused variable `run`

(F841)